src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj

```diff
@@ -39,6 +39,15 @@
     <Reference Include="Microsoft.Spatial, Version=7.8.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\sln\packages\Microsoft.Spatial.7.8.2\lib\net45\Microsoft.Spatial.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.OData.Core, Version=7.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
>Microsoft.OData.Cor [](start = 24, length = 19)

why do you install 7.8.1, since we have 7.8.2 installed?

Reply:
What was the rationale here?

---
In reply to: [604609447](https://github.com/OData/WebApi/pull/2455#discussion_r604609447) [](ancestors = 604609447)

Reply:
I guess it was accidental during some rebase or so !!